### PR TITLE
Cleanup Migrations

### PR DESCRIPTION
**Pull request by conbrad**
- Script to pull fire zones from esri server and import them from files to speed up dev deployments
- Removes redundant migrations: old fuel layer imports and associated combustible fuel type calcs
- Note: since the last migration is removed, we'll likely need to run `stamp` on the previous migration revision so the prod database doesn't complain: https://alembic.sqlalchemy.org/en/latest/api/commands.html#alembic.command.stamp

# Test Links:
[Landing Page](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3625-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: zone_481.geojson [hunk 1]
```diff
@@ -0,0 +1 @@
...5.8932000007, 508968.95720000006], [1490583.8584000003, 510470.6532000005], [1492355.0956999995, 518334.68239999935], [1491308.2809999995, 521066.0795000009], [1495446.6787, 530848.7774999999], [1496152.5711000003, 537408.5092999991], [1500272.3279, 539708.9989999998], [1501591.5471, 549895.0096000005], [1509333.200099999, 547306.9069999997], [1510532.9472000003, 548308.9236999992], [1509936.0902999993, 553217.8772999998], [1512997.4448000006, 556661.7409000006], [1519426.6698000003, 556319.3234999999], [1519304.9940000009, 565655.2257000003], [1522646.9327000007, 565167.1259000003], [1524305.0427, 561652.6063000001], [1527038.6434000004, 561222.5099], [1531073.3099000007, 569889.8015999999], [1535986.6667, 568966.9482000005], [1536642.7149, 570605.8478999995], [1538782.1048000008, 570153.4283000007], [1539002.4859999996, 572229.4941000007], [1546511.2076999992, 573564.3313999996]]]], "attributes": {"MOF_FIRE_ZONE_ID": 481.0, "MOF_FIRE_CENTRE_NAME": "Southeast Fire Centre", "MOF_FIRE_ZONE_NAME": "Boundary Fire Zone", "HEADQUARTERS_CITY_NAME": "Grand Forks", "OBJECTID": 1092, "FEATURE_AREA_SQM": 6590044104.883, "FEATURE_LENGTH_M": 457055.5345, "GEOMETRY.AREA": 0, "GEOMETRY.LEN": 0}}
```
Comment (brettedw) [inline zone_481.geojson:1]: I think these are all currently in the root dir of our repo. 
Also, might not be worth the effort, but all these geojsons could be a single file & iterate through features

Comment (conbrad) [reply zone_481.geojson:1]: Oops thanks I'll move those. I'll defer merging them into a single file for now unless it becomes a problem.

Comment (conbrad) [reply zone_481.geojson:1]: I'll also delete the deployment and crunchydb cluster so it reruns migrations

**Comment by dgboss** [comment]
I don't think we actually need any of fire zone features from the Esri map service. It looks like we delete all the fire zones in 8e85e2b291a9_remove_zone_shapes.py and repopulate the advisory_shapes table with fire zone units in 505a3f03ba75_add_fire_zone_units_to_advisory_shapes.py.

**Comment by conbrad** [comment]
> I don't think we actually need any of fire zone features from the Esri map service. It looks like we delete all the fire zones in 8e85e2b291a9_remove_zone_shapes.py and repopulate the advisory_shapes table with fire zone units in 505a3f03ba75_add_fire_zone_units_to_advisory_shapes.py.

Good eye, I'll move the fire zone units migration to become the earliest fire zone import migration

**Review by dgboss — APPROVED** [review]
Great work!

**Review by brettedw — APPROVED** [review]
Great work, thanks!

**Review by dgboss — APPROVED** [review]

**Pull request merged**